cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBox.java

```diff
@@ -109,6 +109,11 @@ of this software and associated documentation files (the "Software"), to deal
     private final int kKeyboardReturnTypeSend = 2;
     private final int kKeyboardReturnTypeSearch = 3;
     private final int kKeyboardReturnTypeGo = 4;
+    private final int kKeyboardReturnTypeNext = 5;
+
+    public static final int kEndActionUnknown = 0;
+    public static final int kEndActionNext = 1;
+    public static final int kEndActionReturn = 3;
```

Comment:
The value of end actions is begin with 0, why `kEndActionReturn` does not get a value 2 but 3?
Any special considerations?  thanks.